src/api/reports/reportUtils.ts

```diff
@@ -14,15 +17,24 @@ export function runReport(
     case ReportPathsType.aws:
       report = runAwsReport(reportType, query);
       break;
+    case ReportPathsType.awsCloud:
+      report = runAwsCloudReport(reportType, query);
+      break;
     case ReportPathsType.azure:
       report = runAzureReport(reportType, query);
       break;
-    case ReportPathsType.ocpCloud:
-      report = runOcpCloudReport(reportType, query);
-      break;
+    // case ReportPathsType.azureCloud:
+    //   report = runAzureCloudReport(reportType, query);
+    //   break;
     case ReportPathsType.ocp:
       report = runOcpReport(reportType, query);
       break;
+    case ReportPathsType.ocpCloud:
+      report = runOcpCloudReport(reportType, query);
+      break;
+    // case ReportPathsType.ocpUsage:
+    //   report = runOcpUsageReport(reportType, query);
+    //   break;
```

Comment:
should we keep this comment?

Reply:
Merged the comments via a separate PR for the OpenShift Usage overview